library/include/hipsparse.h

```diff
@@ -9197,6 +9199,69 @@ hipsparseStatus_t hipsparseSpGEMM_copy(hipsparseHandle_t      handle,
                                        hipsparseSpGEMMDescr_t spgemmDescr);
 #endif
 
+#if(!defined(CUDART_VERSION) || CUDART_VERSION >= 11000)
```

Comment:
I think spgemm reuse only exists in cusparse 11.3.1 and greater. This will fail to compile with cusparse backend I think